swtpm/Dockerfile.template

```diff
@@ -0,0 +1,5 @@
+ARG BALENA_ARCH=%%BALENA_ARCH%%
+
+FROM balenalib/${BALENA_ARCH}-alpine:3.17-20221210
```

Comment:
Any particular reason for the balenalib image and not official alpine multiarch image?

Asking because the datestamp is not always the same between architectures of the same balenalib image build.

Reply:
@klutchell Originally, I had tried adding `swtpm` to the same image that QEMU runs in. Unfortunately, it's only packaged with a much newer Alpine version, and bumping the version for our existing image isn't straightforward, so I moved it to its own service. I think we could easily switch to the standard Alpine image here.

Reply:
@klutchell Does multiarch work properly for builds now? I remember that being one of the reasons we had balenalib images tagged per architecture.

Reply:
There should be no issues using multiarch base images. Remember that these builds for QEMU tests are just using docker-compose, and not balena tools.

Both core and client are using multiarch base images.

Reply:
Ah, right. For some reason I was thinking this needed to run on testbots too, but that's entirely separate.